src/features/Instructors/InstructorsTable/columns.jsx

```diff
@@ -5,7 +5,7 @@ import { daysWeek, hoursDay } from 'features/constants';
 
 import LinkWithQuery from 'features/Main/LinkWithQuery';
 
-const columns = [
+const getColumns = (showInstructorFeature) => [
```

Comment:
This file can not call `getConfig`?

```javascript
const showInstructorFeature = getConfig().SHOW_INSTRUCTOR_FEATURES || false;


const instructorFeatures = showInstructorFeatures ? [{features}] : [] 

const colums = [
 //items,
...instructorFeatures
]
``` 

Event better, maybe in the future that flag could be useful in other part of the app:
```javascript
// constants.js
export const SHOW_INSTRUCTOR_FEATURES = getConfig().SHOW_INSTRUCTOR_FEATURES || false;
``` 

```javascript
// colums.js
import { SHOW_INSTRUCTOR_FEATURES } from 'constants'

const instructorFeatures = SHOW_INSTRUCTOR_FEATURES ? [{features}] : [] 

const colums = [
 //items,
...instructorFeatures
]```

Reply:
@01001110J  I got an error when trying to define in columns.jsx or constants. In the first render, it doesn't take the value defined in settings, only the default value, so the column never renders. I even tried removing the useMemo, but I get the same result. So it's not a possible approach.